admin_runs.go

```diff
@@ -0,0 +1,125 @@
+package tfe
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+	"time"
+)
+
+// Compile-time proof of interface implementation.
+var _ AdminRuns = (*adminRuns)(nil)
+
+// AdminRuns Users Admin API contains endpoints to help site administrators manage
+// user accounts.
+//
+// TFE API docs: https://www.terraform.io/docs/cloud/api/admin runs.html
+type AdminRuns interface {
+	// List all the runs of the given installation.
+	List(ctx context.Context, options AdminRunsListOptions) (*AdminRunsList, error)
+
+	// Force-cancel a run by its ID.
+	ForceCancel(ctx context.Context, runID string, options RunForceCancelOptions) error
+}
+
+// runs implements Users.
+type adminRuns struct {
+	client *Client
+}
+
+type adminRun struct {
+	ID                     string               `jsonapi:"primary,runs"`
+	Actions                *RunActions          `jsonapi:"attr,actions"`
+	CreatedAt              time.Time            `jsonapi:"attr,created-at,iso8601"`
+	ForceCancelAvailableAt time.Time            `jsonapi:"attr,force-cancel-available-at,iso8601"`
+	HasChanges             bool                 `jsonapi:"attr,has-changes"`
+	IsDestroy              bool                 `jsonapi:"attr,is-destroy"`
+	Message                string               `jsonapi:"attr,message"`
+	Permissions            *RunPermissions      `jsonapi:"attr,permissions"`
+	PositionInQueue        int                  `jsonapi:"attr,position-in-queue"`
+	Source                 RunSource            `jsonapi:"attr,source"`
+	Status                 RunStatus            `jsonapi:"attr,status"`
+	StatusTimestamps       *RunStatusTimestamps `jsonapi:"attr,status-timestamps"`
+
+	// Relations
+	Workspace    *Workspace    `jsonapi:"relation,workspace"`
+	Organization *Organization `jsonapi:"relation,workspace.organization"`
+}
+
+// AdminRunsList represents a list of runs.
+type AdminRunsList struct {
+	*Pagination
+	Items []*Run
+}
+
+// AdminRunsListOptions represents the options for listing runs.
+type AdminRunsListOptions struct {
+	ListOptions
+	RunStatus *string `url:"filter[status]"`
+	Query     *string `url:"q"`
+	Include   string  `url:"include"`
+}
+
+func (o AdminRunsListOptions) valid() error {
+	if *o.RunStatus != "" {
```

Comment:
I consistently get segfault panics on this line. I am pretty sure you need to check that RunStatus != nil here.